extensions/ext/GL_EXT_vulkan_glsl_relaxed.txt

```diff
@@ -0,0 +1,177 @@
+Name
+
+    EXT_vulkan_glsl_relaxed
+
+Name Strings
+
+    GL_EXT_vulkan_glsl_relaxed
+
+Contact
+
+    Malcolm Bechard (malcolm 'at' derivative.ca), Derivative
+
+Contributors
+
+    William Brown, (will.brown039 'at' gmail.com), Derivative 
+    Malcolm Bechard, Derivative
+
+Notice
+
+    Copyright (c) 2020 The Khronos Group Inc. Copyright terms at
+        http://www.khronos.org/registry/speccopyright.html
+
+Status
+
+Version
+
+Number
+
+    TBD.
+
+Dependencies
+
+    This extension requires GL_KHR_vulkan_glsl.
+    
+    This extension can be applied to OpenGL GLSL versions 1.40
+    (#version 140) and higher.
+
+    This extension can be applied to OpenGL ES ESSL versions 3.10
+    (#version 310) and higher.
+
+    All these versions map GLSL/ESSL semantics to the same SPIR-V 1.0
+    semantics (approximating the most recent versions of GLSL/ESSL).
+
```

Comment:
There should also be a paragraph about what versions of what specifications this is written against.  This is usually a quite recent specification version, and independent of the statements of how back the extension can be applied.

Reply:
Just so I'm clear. Are you saying specify which version of GL_KHR_vulkan_glsl the spec is against, or also the GLSL version?

Reply:
It depends what specification you are writing it against.

Given that this has been incorporated into the GLSL specification, I think it's best to write against that:  The OpenGL Shading Language, Version 4.60.7.

Then, each normative statement is about what to change in that spec.

Reply:
Since this extension relies heavily on the changes in GL_KHR_vulkan_glsl, is there an example (another extension somewhere that does similar) of how to reconcile both the changes that this extension wants from GL_KHR_vulkan_glsl as well as the changes it reverses.
Should I basically copy/paste all the change-entries from GL_KHR_vulkan_glsl and then edit them to be what they would be when both this extension and GL_KHR_vulkan_glsl are active? That way this extension includes both edits, or is there a different way of going about formatting this?

Reply:
I recommend writing relative to "The OpenGL Shading Language, Version 4.60.7."

You would just say what edits should be made there, and independently state "where the modified features also exist in previous versions, these changes are also supported in previous versions".

Reply:
GL_EXT_vulkan_glsl itself is an example of how to do that.

Reply:
GL_EXT_vulkan_glsl does not contain any paragraph stating what versions of what specification it's written against though, no? Can you be specific about a section from GL_EXT_vulkan_glsl whose formatting/writing style this extension is failing to follow?

Sorry, I'm really struggling to understand what you are asking me to do here. Other than adding a line stating "This was written against The OpenGL Shading Language, Version 4.60.7", I'm unclear what you would like me to edit, and how it should be edited.

I'll add that paragraph, but if there is more I'm missing maybe can you could do a single example edit to this spec to show what you are looking for? Thanks and sorry for not understanding. Thanks

Reply:
Took another stab at how the edits are formatted, and went through the 4.60.7 spec to find other spots that need to be edited by this spec. Hoping this is more in-line with what you are thinking.